config/boards/armsom-sige7.csc

```diff
@@ -0,0 +1,40 @@
+# Rockchip RK3588 SoC octa core 8-64GB SoC 2*2.5GBe eMMC USB3 NvME WIFI
+BOARD_NAME="ArmSoM Sige7"
+BOARDFAMILY="rockchip-rk3588"
+BOARD_MAINTAINER="Z-Keven"
```

Comment:
Please, leave some contacts [here](https://www.armbian.com/update-data/) so we have alternative way of communication.